### PR TITLE
add Sendinblue support

### PR DESCRIPTION
**Pull request by danielegallingani**
here the sendinblue doc: https://apidocs.sendinblue.com/tutorial-sending-transactional-email/

### CODE REVIEW

File: src/main/scala/mailo/http/SendinblueClient.scala [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.Source
+import akka.actor.ActorSystem
+
+import scala.concurrent.{Future, ExecutionContext}
+import javax.mail.Message.RecipientType
+import javax.mail.internet.MimeMessage
+
+import com.typesafe.config.{Config, ConfigFactory}
```
Comment (calippo) [inline src/main/scala/mailo/http/SendinblueClient.scala]: @danielegallingani no beh importale due dipendenze

File: src/main/scala/mailo/http/SendinblueClient.scala [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  def sendMime(
+    message: MimeMessage,
+    tags: List[String] = List.empty,
+    attachments: List[Attachment] = List.empty,
+    headers: Map[String, String] = Map.empty
+  )(
+    implicit
+    executionContext: ExecutionContext
+  ): Future[Either[MailError, MailResponse]] =
+    throw new Exception("unable to send mime messages in Sendinblue")
```
Comment (calippo) [inline src/main/scala/mailo/http/SendinblueClient.scala]: I would throw a `UnsupportedOperationException` instead

File: src/main/scala/mailo/http/SendinblueClient.scala [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    bcc: Option[String],
+    subject: String,
+    content: MailRefinedContent,
+    attachments: List[Attachment],
+    tags: List[String],
+    headers: Map[String, String]
+  )(implicit ec: ExecutionContext): Future[java.util.HashMap[String, Any]] = Future {
+    import mailo.MailRefinedContent._
+
+    val data = new java.util.HashMap[String, Any]()
```
Comment (calippo) [inline src/main/scala/mailo/http/SendinblueClient.scala:104]: (╯°□°）╯︵ ┻━┻

**Review by calippo — COMMENTED** [review]
The library you're using have 2 stars.
I would have used this library that has 13 stars 😂https://github.com/mailin-api/mailin-api-java (but is supported by sendinblue).

I believe that implementing the http client using akka http and directly sendinblue rest api would have been easier :P.

**Comment by danielegallingani** [comment]
@calippo addressed you comments could you please merge&publish? <3

**Pull request merged**